### DISCUSSION TITLE
Dasboard is missing kanban cards for tasks (issues).

### DISCUSSION THREAD

**Discussion by hkdemiralp** [jgraichen/redmine_dashboard#514] [Q&A]
I installed "Redmine Dashboard 2" like specified on the project readme page. But redmine does not display any kanban cards in any grouping options on View menu (card, compact, no grouping, tracker, etc.). I enabled the the plugin both in Redmine settings and Project settings. In the image you can see there are 3 issues in Test sub project, but there are no cards representing them. Why the kanban cards are not displayed on dashboard? The issues have priority (normal) and statuses (new, doing). Any idea?
Info:

Environment:
Redmine version                5.1.2.stable
Ruby version                   3.2.0-p0 (2022-12-25) [x86_64-linux]
Rails version                  6.1.7.7
Environment                    production
Database adapter               Mysql2
Mailer queue                   ActiveJob::QueueAdapters::SidekiqAdapter
Mailer delivery                smtp
Redmine settings:
Redmine theme                  Classic
SCM:
Git                            2.43.0
Filesystem

Redmine plugins: redmine_dashboard              2.15.0

OS: RHEL 9.4

Default administrator account changed 	✔️
Attachments directory writable 	✔️
Plugin assets directory writable (./public/plugin_assets) 	✔️
All database migrations have been run 	✔️
MiniMagick available (optional) 	✔️
ImageMagick convert available (optional) 	✔️
ImageMagick PDF support available (optional) 	✔️
Default queue adapter which is well suited only for dev/test changed✔️
![redmine_kanban_cards_prj](https://github.com/jgraichen/redmine_dashboard/assets/90977461/dc28457e-2731-4357-bcbc-3c99929f30e3)

**Comment by olympus22git** [comment]
I also hit the same issue. Below is my installed version information. Any one has the same problem get resolved?

Environment:
  Redmine version                5.1.3.stable
  Ruby version                   3.2.4-p170 (2024-04-23) [x86_64-linux]
  Rails version                  6.1.7.8
  Environment                    production
  Database adapter               Mysql2
  Mailer queue                   ActiveJob::QueueAdapters::AsyncAdapter
  Mailer delivery                smtp
Redmine settings:
  Redmine theme                  Fluentmine
SCM:
  Subversion                     1.14.2
  Mercurial                      6.3.2
  Bazaar                         3.3.2
  Git                            2.39.2
  Filesystem                     
Redmine plugins:
  redmine_dashboard              2.15.0

**Comment by jgraichen** [comment]
> In the image you can see there are 3 issues in Test sub project, but there are no cards representing them. Why the kanban cards are not displayed on dashboard? The issues have priority (normal) and statuses (new, doing). Any idea? Info:

It seems you have collapsed the swim lane content. Please try to expand these:

[Kooha-2024-06-19-11-20-56.webm](https://github.com/jgraichen/redmine_dashboard/assets/800848/58add1c4-622d-4fed-8785-d512e86a9476)

**Comment by hkdemiralp** [comment]
After I created a workflow, the kanban cards, and tasks showed up. In my screenshot, there were no titles like "New, In Progress, Resolved, etc". These titles also started showing. I am a new Redmine and Redmine Dashboard 2 user, and this issue was confusing. It would be great if in the front page  "Configure Redmine" section, it is stated that the workflow need to be edited/adjusted to see the kanban cards and tasks. Or possible it would be better if editing workflow is not needed for the kanban board to show up.  Since @olympus22git has not confirmed if the issue has been also resolved for him, I will not close the issue, yet.

**Comment by olympus22git** [comment]
After giving more tries yesterday, I found the reason. This may not be an issue after I enable the proper permission in workflow as shown below. The dashboard shows all of the statuses. I think this problem will only appear when the first time install the Redmine and plug-in without setting anything, then it will have the problem.

![redmine_workflow](https://github.com/jgraichen/redmine_dashboard/assets/26355208/79b49509-d58d-4922-a06b-fba581868c02)

**Comment by hkdemiralp** [comment]
Hi, @olympus22git, nice to see our problem solved.
Yes, the problem only appears if the default workflow permissions are not changed after installing Redmine. I could never guess the default permissions need to be changed at first. So stating this in the "Configure Redmine" section will save time for both devs and new users. I am closing this bug report.
Thank you very much @olympus22git and @jgraichen for involving in this report.

**Reply by jgraichen** [reply]
Hi @hkdemiralp, the plugin shows all issues the current user can see and allows drag-n-drop to columns the user is allowed to move tickets too.

The plugin is automatically tested with the latest Redmine versions, and should work with the default data from Redmine.

Do you have details and/or screenshots of how your workflows looked before and how that didn't work?

**Reply by hkdemiralp** [reply]
Hi @jgraichen, for comparison if you look at my first post the dasboard image has no kanban tasks. And now the same dashboard page has the yellow kanban tasks with the same view options in the image below. In the image you can see that the "Sistem" project has two kanban tasks and the Genel project has one kanban task. (I renamed test project to "Sistem", "Subproject 1" to "Genel" and I also deleted "subproject 2".) Now whatever view options I choose, I can see the yellow kanban tasks. 
Also, I can now see the issue statuses "Yeni, Yapılıyor, Çözüldü, Geribildirim, Done" (New, In Progress, Resolved, Feedback, Done).
![redmine_works](https://github.com/jgraichen/redmine_dashboard/assets/90977461/9a42020c-991e-4c0e-8344-abcfa0653492)
